packages/SwingSet/src/kernel/kernel.js

```diff
@@ -384,8 +394,14 @@ export default function buildKernel(
       kernelDelivery,
       vatDelivery,
     );
+    // Ensure that the vatSlogger is available before clist translation.
+    kernelSlog.provideVatSlogger(vatID);
```

Comment:
I suspect this line can be removed: if `provideVatSlogger` can create vatsloggers on-demand, then whoever needs it later should be doing their own call, and there's no need to "warm up" here. But I'm happy to see that happen in a follow-on PR.